src/extension.ts

```diff
@@ -109,7 +109,8 @@ function list(yo: Yeoman): Promise<QuickPickItem[]> {
 			yo.getEnvironment().lookup(() => {
 				const generators = yo.getGenerators().map(generator => {
 					return {
-						label: generator.name.split(/\-(.+)?/)[1],
+						// fix about generators started with '@' scope
+						label: generator.name.replace(/generator\-/i, ''),
```

Comment:
Change the the replacement to this one

```js
.replace(/(^|\/)generator\-/i, '$1')
```

We don't want `foo-generator-foo` to match. Only if it starts at the beginning (`^`) or after a `/` which indicates a namespace.